packages/server/providers/pouchdb.js

```diff
@@ -139,6 +138,26 @@ const pouchDb = (app, io, pouchOptions) => {
       }).catch(() => res.send([]))
   })
 
+  app.get('/:wargame/last', (req, res) => {
+    const databaseName = checkSqliteExists(req.params.wargame)
+
+    if (!databaseName) {
+      res.status(404).send({ msg: 'Wrong Wargame Name', data: null })
+    }
+
+    const db = new PouchDB(databaseName, pouchOptions)
+
+    db.find({
+      selector: {
+        messageType: INFO_MESSAGE,
+        _id: { $ne: wargameSettings }
+      },
+      limit: 1,
+      sort: [{ _id: 'desc' }]
```

Comment:
hey @lilitkarapetyan  - this is clever :-)

But, I have these worries:
1.  Doesn't PouchDb have to sort all the documents by `_id` before it can determine the last one?
2. Wouldn't `initial_wargame` always come **after** a date-time like `2022-03-18T17:08:22.008Z` ?

Reply:
Hi Ian, From my research I found that the PouchDb doesn't give us to take last one, the best solution in this case just make sort and then take.